app/views/drawings/index.html.haml

```diff
@@ -1,5 +1,2 @@
 #drawings
   = render 'drawings'
-
-#paginator.text-center
-  = link_to_next_page @drawings, 'MORE', remote: true, id: 'load_more'
```

Comment:
Ok cool, I'll put it back and make the purple blob invisible and can look at the JS next weekend :)